extension/intents/clipboard/clipboard.toml

```diff
@@ -125,3 +125,9 @@ match = """
 
 [[clipboard.paste.example]]
 phrase = "Paste"
+
+[clipboard.copyValue]
+description = "Copies the value given"
+match = """
+  copy [value]
```

Comment:
This feels like it might catch some things that it's not intended to. Like, "copy the current page's URL" (to think of a phrase we probably don't parse).

I think it should be explicit, like `copy text [value]`